### PR TITLE
lancelot: update to v10

### PR DESCRIPTION
**Pull request by hipexscape**

### CODE REVIEW

File: lancelot.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "acex88",
+			"oem": "Redmi",
+			"device": "Redmi 9",
+			"filename": "crDroidAndroid-14.0-20231221-lancelot-v10.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/lancelot/10.x/crDroidAndroid-14.0-20231221-lancelot-v10.0-BETA.zip/download",
```
Comment (gwolf2u) [inline lancelot.json:8]: upload file

File: lancelot.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			"timestamp": 1703154129,
+			"md5": "7c762ceffb83af19e257533cc5686aba",
+			"sha256": "ebb2295e09bf168e735e1d112e5c72743fa38c487cfdbd36b76a9bfacac71437",
+			"size": 1123150233,
+			"version": "10.0",
+			"buildtype": "Monthly",
+			"forum": "",
+			"gapps": "",
```
Comment (gwolf2u) [inline lancelot.json]: no agpps?
maybe set https://nikgapps.com/crdroid-official

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: lancelot.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+			"device": "Redmi 9",
+			"filename": "crDroidAndroid-14.0-20231221-lancelot-v10.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/lancelot/10.x/crDroidAndroid-14.0-20231221-lancelot-v10.0-BETA.zip/download",
+			"timestamp": 1703154129,
+			"md5": "7c762ceffb83af19e257533cc5686aba",
+			"sha256": "ebb2295e09bf168e735e1d112e5c72743fa38c487cfdbd36b76a9bfacac71437",
+			"size": 1123150233,
+			"version": "10.0",
+			"buildtype": "Monthly",
+			"forum": "",
```
Comment (ghost) [inline lancelot.json]: Missing forum.

Comment (gwolf2u) [reply lancelot.json]: ehh... thought I commented that also, but seems I didn't
nice catch 👍🏼

File: lancelot.json (hunk 2 above)
Comment (hipexscape) [reply lancelot.json]: Ok I will do that

File: lancelot.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+			"device": "Redmi 9",
+			"filename": "crDroidAndroid-14.0-20231221-lancelot-v10.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/lancelot/10.x/crDroidAndroid-14.0-20231221-lancelot-v10.0-BETA.zip/download",
+			"timestamp": 1703154129,
+			"md5": "7c762ceffb83af19e257533cc5686aba",
+			"sha256": "ebb2295e09bf168e735e1d112e5c72743fa38c487cfdbd36b76a9bfacac71437",
+			"size": 1123150233,
+			"version": "10.0",
+			"buildtype": "Monthly",
+			"forum": "https://xdaforums.com/t/rom-14-official-crdroid-v10-aosp-stable.4625907/",
```
Comment (gwolf2u) [inline lancelot.json:15]: do make sure to change download page url on forum, to link to site

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**